### ISSUE TITLE
No 'calt' specified, yet: OpenType feature 'calt' not available for 'Helvetica'

### ISSUE THREAD

**Issue by justvanrossum** [typemytype/drawbot#120]
When using `txt.appendGlyph()`:

```python
txt = FormattedString()
txt.append("abc", font="Helvetica", fontSize=20, fill=(1, 0, 0))
txt.appendGlyph("dollar")
text(txt, (10, 50))
```

```
*** DrawBot warning: OpenType feature 'calt' not available for 'Helvetica' ***
```

(Found via #119)

**Comment by typemytype** [comment]
Mmm, maybe all those warning should become silent again

See https://github.com/typemytype/drawbot/blob/master/drawBot/context/baseContext.py#L1484

**Comment by typemytype** [comment]
fixed 7a928506eec5e79aa5b6b2e774e80a6da8f9416f

**Comment by justvanrossum** [comment]
Thanks!

(If you write `"fixes #120"` in your commit message, the issue closes automatically, and has the additional benefit that the commit then nicely references the issue. WinWin!)

**Comment by typemytype** [comment]
oh thats nice, didnt know that!!
